Camera2Basic/app/src/main/java/com/example/android/camera2/basic/fragments/CameraFragment.kt

```diff
@@ -147,6 +147,11 @@ class CameraFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
         overlay = view.findViewById(R.id.overlay)
         viewFinder = view.findViewById(R.id.view_finder)
+        capture_button.setOnApplyWindowInsetsListener { v, insets ->
+            v.translationX = (-insets.systemWindowInsetRight).toFloat()
+            v.translationY = (-insets.systemWindowInsetBottom).toFloat()
```

Comment:
Is there a way to increase the padding instead of applying a different UI transformation?

Reply:
This is a conscious choice. Translation, unlike padding, does not require or start layout recalculation. At any rate, why do you speak about "increase padding"? Currently, there is no padding specified for the button.

Reply:
I thought the button already had defined padding, but apparently I was misremembering. The logic was to not apply more than one type of transformation to the same UI element if possible, but since no other transformation is currently being applied this is probably fine.